### ISSUE TITLE
PHPMD complains about itself

### ISSUE THREAD

**Issue by morganestes** [phpmd/phpmd#77]
I just installed PHPMD in [PhpStorm 6.0](https://www.jetbrains.com/phpstorm/webhelp/using-php-mess-detector.html)  and ran it for the first time in my project. The very first warning was about itself and not my code; I can't figure out how to to keep it from checking its own files and checking mine instead.

> phpmd: PHP Warning:  date(): It is not safe to rely on the system's timezone settings. You are _required_ to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected 'America/Chicago' for 'CDT/-5.0/DST' instead in [/usr/lib/share/pear/PHP/PMD/Renderer/XMLRenderer.php on line 97](https://github.com/phpmd/phpmd/blob/master/src/main/php/PHP/PMD/Renderer/XMLRenderer.php#L97)

**Comment by bgaillard** [comment]
Hi, we also have this issue using PHP 5.6. 
```
PHP Warning:  date_default_timezone_get(): It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected the timezone 'UTC' for now, but please set date.timezone to select your timezone. in /home/ec2-user/.jenkins/jobs/fatcap-php-services-testing/workspace/vendor/phpmd/phpmd/src/bin/phpmd on line 24
PHP Stack trace:
PHP   1. {main}() /home/ec2-user/.jenkins/jobs/fatcap-php-services-testing/workspace/vendor/phpmd/phpmd/src/bin/phpmd:0
PHP   2. date_default_timezone_get() /home/ec2-user/.jenkins/jobs/fatcap-php-services-testing/workspace/vendor/phpmd/phpmd/src/bin/phpmd:24
```

The fix applied is not valid for PHP 5.6, the warning is explicit, even if the system timezone is set we should set a timezone explicitly in the source code. 

So (at least for PHP 5.6) we should replace ...
```php
if (!ini_get('date.timezone') && !date_default_timezone_get()) {
    date_default_timezone_set('UTC');
}
```

by ... 

```php
if (version_compare(phpversion(), '5.6.0', '>=') ||  (!ini_get('date.timezone') && !date_default_timezone_get())) {
    date_default_timezone_set('UTC');
}
```

**Comment by michaelhogg** [comment]
@hbandura: Yes, even after @manuelpichler's fix, a timezone warning is still raised.

I've just created PR #297 which completely fixes the issue (@bgaillard: even on PHP 5.6).
